lambda/notifications.tf

```diff
@@ -97,6 +105,26 @@ data "aws_ssm_parameter" "gov_uk_transfer_complete_template_id" {
   name  = "/${local.environment}/gov_uk_notify/transfer_complete_template_id"
 }
 
+data "aws_ssm_parameter" "gov_uk_metadata_review_tb_template_id" {
```

Comment:
Do we need multiple different templates, rather than using a single template and pass in params into it to construct the correct message.

Reply:
The email templates are quite different from one another so I'm not sure how we would do that. You can see the templates I've created here https://www.notifications.service.gov.uk/services/47204260-cc6b-4814-ad27-ec83bec97003/templates/all

Reply:
OK. I'm just a bit concerned about long term maintenance of templates.